brianlong/www.validators.app
# 183882630 validator details vue (#683)

#### What's this PR do?
It's a feature brach for two tasks:
- re-write validator's details to Vue https://www.pivotaltracker.com/story/show/183882630
- update validator charts https://www.pivotaltracker.com/story/show/184193352

#### How should this be manually tested?
- Choose any validator from the list, click on the link to open validator details
- Compare validator details page against production - everything should look the same or better

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183882630)
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184193352)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
